Fonts.css

```diff
@@ -0,0 +1,344 @@
+
+
+h1 {
+  font-family: Arial;
+  font-size: 40px;
+}
+h2 {
+  font-family: Martial;
+  font-size: 24px;
+}
+header {
+  padding: 30px;
+  background-color:  red;
+
+}
+body {
+  font-family: Arial, Helvetica, sans-serif;
+}
+.box1{
+
+  height: 100px;
+  width: 100%;
+  background-color:  #f0f0f5;
+
+}
+
+.grid-box2{
+    display: grid;
+    padding-top: 40px;
+    padding-bottom: 10px;
+    grid-template-columns: 400px 400px;
+    padding-right: 300px;
+    justify-content: center;
+    min-height: 500px;
+    grid-gap: 50px;
+  }
+
+.box2{
+
+  height: 400px;
+  width: 100%;
+  background-color: pink;
+
+}
+
+.box2-head1{
+  width: auto;
+  height: auto;
+  grid-column: 1 / 2;
+  grid-row: 1/2;
+}
+
+.box2-head2{
+  width: 750px;
+  height: auto;
+  grid-column: 2 / 3;
+  grid-row: 1/2;
+}
+.box1-about{
+
+  height: 600px;
+  width: 100%;
+}
+.head1-about{
+  float:left;
+  width: 50%;
+
+}
+
+.head2-about{
+  width: 50%;
+
+}
+
+.container{
+
+  height: 500px;
+}
+.container2{
+
+  height: 600px;
+}
+.head1{
+  width: auto;
+  grid-column: 1 / 2;
+  grid-row: 1/2;
+}
+
+.head2{
+  width: auto;
+  grid-column: 2 / 3;
+  grid-row: 1/2;
+}
+.head3{
+  width: auto;
+  grid-column: 3 / 4;
+  grid-row: 1/2;
+}
+
+.head1-header{
+  width: 300px;
+  height: 150px;
+  display: flex;
+  grid-column: 1 / 2;
+  grid-row: 1/2;
+}
+.head1-footer{
+  width: auto;
+  height: auto;
+  display: flex;
+justify-content: center;
+}
+.head2-header{
+  width: 300px;
+  height: 150px;
+  display: flex;
+  grid-column: 3 / 4;
+  grid-row: 1/2;
+}
+
+.head3{
+  display: flex;
+  width: auto;
+  height: 150px;
+  justify-content: flex-start;
+  grid-column: 3 / 4;
+  grid-row: 1/2;
+}
+.head3-header{
+  width: 10%;
+  float:left;
+}
+.head3-lessons{
+  grid-column: 3 / 4;
+  grid-row: 1/2;
+
+  width: auto;
+}
+.head4-lessons{
+  grid-column: 4 / 5;
+  grid-row: 1/2;
+
+  width: auto;
+}
+.head5-lessons{
+  grid-column: 5 / 6;
+  grid-row: 1/2;
+
+  width: auto;
+}
+.container-instructor{
+
+  height: 700px;
+  width: 100%;
+}
+.head1-instructor{
+  width: auto;
+  grid-column: 1 / 2;
+  grid-row: 1/2;
+height: auto;
+}
+
+.head2-instructor{
+  width: auto;
+  grid-column: 2 / 3;
+  grid-row: 1/2;
+  height: auto;
+
+}
+
+.head3-instructor{
+  display: block;
+  width: auto;
+  grid-column: 3 / 4;
+  grid-row: 1/2;
+  height: auto;
+
+}
+
+footer {
+  background-color: #777;
+  padding: 10px;
+  text-align: center;
+  color: white;
+  background-image: url(footerImage.jpg);
```

Comment:
it's generally a good idea to keep your images in a folder then make a relative path to them. `background-image: url('./images/footerImage.jpg');`